djongo/sql2mongo/functions.py

```diff
@@ -19,12 +20,22 @@ def token2sql(token: Token,
                   query: 'query_module.BaseQuery'
                   ) -> U['CountFunc',
                          'SimpleFunc']:
+        ## FIX: agg(distinct)
+        if isinstance(token[0], Identifier) and token[1][1].match(tokens.Keyword, 'DISTINCT'):
+            token.is_agg_distinct = True
+        else:
+            token.is_agg_distinct = False
```

Comment:
if/else redundant:
token.is_agg_distinct = isinstance(token[0], Identifier) and token[1][1].match(tokens.Keyword, 'DISTINCT')